### ISSUE TITLE
Remote-Containers: No easy way to change the open directory

### ISSUE THREAD

**Issue by HughxDev** [microsoft/vscode-remote-release#6706]
Issue Type: <b>Feature Request</b>

If I attach to a container, then open a specific directory in the container, I cannot change it to a different one, even if I exit the window and re-attach. There is an option to “open a folder inside the container” but it brings me to the system dialog for my host, which is not what I want. (In fact, I don’t even understand what this is supposed to do.)

VS Code version: Code 1.67.0 (57fd6d0195bb9b9d1b49f6da5db789060795de47, 2022-05-04T12:14:47.100Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No
Remote OS version: Linux x64 5.10.104-linuxkit

**Comment by chrmarti** [comment]
`File` > `Open Folder...` should work when you are already connected to a container. What happens when you do that? (Where do you see the “open a folder inside the container” option? Could you attach a screenshot?)

**Comment by HughxDev** [comment]
@chrmarti `File > Open Folder...` does work, thanks. I just never thought to do that because I figured it was only useful for opening a folder on my local filesystem. I never would have expected the behavior of a menu item that rudimentary to change. And since Remote Containers is a plugin, I assumed I needed to run a specific Remote Containers command to change the default folder that opens when attaching to the container. That is where I found the `Open Folder in Container` option. Here is a screenshot, it’s available after doing `Cmd + Shift + P`.

<img width="880" alt="open folder in container" src="https://user-images.githubusercontent.com/229232/167969748-0ffc285d-754c-4266-a8c9-2d9283f4e25d.png">

When I select this I get my local filesystem dialog, which is again the inverse of my expectation, because this label to me means “open a folder existing within the container already”, not “take a folder on your local filesystem and open it in a new container”.
